cli/smartcontract/generate.go

```diff
@@ -0,0 +1,348 @@
+package smartcontract
```

Comment:
CLI should just wrap our internal libraries into something user-facing, but this functionality might be interesting to have in a library too.

Reply:
Fixed.